### ISSUE TITLE
Move the post-query processing jobs to a dedicated module

### ISSUE THREAD

**Issue by sorsaffari** [vaticle/typedb-studio#186]
Currently, the pot-processing that takes place after running queries, is done outside the scope of the `CanvasDataBuider`. Because of this, the `node` objects produced by `CanvasDataBuider` must include properties of the original Graql Concepts.

Ideally, node objects should only contain properties expected by Visjs ~~and therefore the post-processing job should be moved to the `CanvasDataBuider`~~

**Comment by marco-scoppetta** [comment]
As for our discussion, an update on this:

`computeAttributes()` (or another name) should stay as a separate method, but it should accept `Answer`s or `Concept`s as parameters, so that building canvas nodes and calling methods on Concepts remain 2 different things working on 2 different type of objects.

Pseudo code for `computeAttributes(answers)`:
```
/**
* Given an array of answers return an array of Objects { id, attributes}
*/
function computeAttributes(answers){
  const concepts = extractConceptsFromAnserws(answers); // it can use concepts directly as a parameter, this is just an hypothesis
  const instances = concepts.filter(c => c.isThing());
  return await Promise.all(instances.map(loadAttributes(instance)));
}

"private" async function loadAttributes(instance){
   const attributes = await (await node.attributes()).collect();
   const attributesWithValues = await attributesWithValues(attributes);
   return Object.assign({id: instance.id , attributes });
}
// This function can be rewritten in a more elegant fashion, but which will be less performant, 
// up to us to decide whether prioritise elegance or performance
"private" function attributesWithValues(attributes){
   return Promise.all(attributes.map(async (concept) => {
      const attribute = {};
        await Promise.all([
          concept.type().then(type => type.label()).then((label) => { attribute.type = label; }),
          concept.value().then((value) => { attribute.value = value; }),
        ]);
      return attribute;
    }))
}
```

**Comment by marco-scoppetta** [comment]
Once this is done let's remember to remove `txService` from the nodes when we build them!

**Comment by alexjpwalker** [comment]
This issue is no longer relevant as Studio uses new architecture.
